src/main/java/teammates/common/datatransfer/attributes/SessionAttributes.java

```diff
@@ -19,20 +19,41 @@
      */
     Comparator<SessionAttributes> ASCENDING_ORDER = (session1, session2) -> {
 
+        FeedbackSessionAttributes feedbackSession1 = null;
+        FeedbackSessionAttributes feedbackSession2 = null;
+        Boolean isSession1PrivateFeedback = false;
+        Boolean isSession2PrivateFeedback = false;
+        if (session1 instanceof FeedbackSessionAttributes) {
+            feedbackSession1 = (FeedbackSessionAttributes) session1;
+        }
```

Comment:
Understand that you're following the example of `DESCENDING_ORDER`, but a better way would be to add `isPrivateSession()` to the interface for much more concise code. Can update `DESCENDING_ORDER` too.